.github/scripts/generate_changelog_md.sh

```diff
@@ -35,19 +57,31 @@ CONTENT=$(
 
 CONTENT=$(jq '.choices[0].message.content' <<< $CONTENT)
 
+# exit if the content is null
+if [ "$CONTENT" == "null" ]; then
+  echo "Failed to generate changelog content."
+  echo "CHANGELOG_GENERATED=0" >> $GITHUB_ENV
+  exit 1
+fi
+
 # Create directory content/changelogs if it doesn't exist
 mkdir -p content/changelogs
 
 # Create a new file with the current release as the name
 echo ${CONTENT//\"/} > content/changelogs/$CURRENT_RELEASE.md
+SHORT_COMMIT_LAST_RELEASE=$(echo $COMMIT_LAST_RELEASE | cut -c 1-5)
+SHORT_COMMIT_CURRENT_RELEASE=$(echo $COMMIT_CURRENT_RELEASE | cut -c 1-5)
 
 # Add the full changelog on the end of the file
 echo """
-#### **Full Changelog:** [$LAST_RELEASE...$CURRENT_RELEASE](https://github.com/mediar-ai/screenpipe/compare/$LAST_RELEASE...$CURRENT_RELEASE)
+#### **Full Changelog:** [$SHORT_COMMIT_LAST_RELEASE..$SHORT_COMMIT_CURRENT_RELEASE](https://github.com/mediar-ai/screenpipe/compare/$SHORT_COMMIT_LAST_RELEASE..$SHORT_COMMIT_CURRENT_RELEASE)
 """ >> content/changelogs/$CURRENT_RELEASE.md
```

Comment:
This works but note that if have multiple versions, it'll work fine but the specific changelog for that version will be overwritten. is that ok @louis030195?

if we need to keep all changelogs from all versions – even duplicates – it'll require a few tweaks but I can implement it. I'll basically add timestamp to the changelog name, so we can differentiate between them. if it's ok override the duplicated version, feel free to merge it (unless you have any comments)